torch/csrc/autograd/python_function.cpp

```diff
@@ -559,6 +559,16 @@ static void _trace_post_record(
   }
 
   node->i_(jit::attr::inplace, is_inplace);
+  if (node) {
+    auto module_name = PyDict_GetItemString(((PyTypeObject*)op_obj)->tp_dict, "__module__");
+    if (module_name) {
```

Comment:
nit: more compat code, same with `ptr` below.
```suggestion
    if (auto module_name = PyDict_GetItemString(((PyTypeObject*)op_obj)->tp_dict, "__module__")) {
```